### PR TITLE
[@types/cytoscape] Update types to align with 3.19

### PR DESCRIPTION
**Pull request by Nadhum**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

Changes:
Fix cy.viewport options https://js.cytoscape.org/#cy.viewport
Update background-image styles https://js.cytoscape.org/#style/background-image
Deprecate PieChart https://github.com/cytoscape/cytoscape.js/issues/2789
Add line-opacity to EdgeLine http://js.cytoscape.org/#style/edge-line
Add LoopEdges https://js.cytoscape.org/#style/loop-edges
Add TaxiEdges https://js.cytoscape.org/#style/taxi-edges
Add circle-triangle to ArrowShape
Update labels styles http://js.cytoscape.org/#style/labels
Remove Shadow Styles https://github.com/cytoscape/cytoscape.js/issues/1758
Update CollectionEvent and GraphEvent

### CODE REVIEW

File: types/cytoscape/index.d.ts [hunk 1]
```diff
@@ -4095,7 +4194,7 @@ declare namespace cytoscape {
              * distance perpendicular to a line formed
              * from source to target, e.g. -20 20 - 20.
              */
-            'control-point-distances': PropertyValueEdge<string>;
+            'control-point-distances': PropertyValueEdge<number | number[] | string>;
```
Comment (Nadhum) [inline types/cytoscape/index.d.ts:4197]: Not sure if string should be kept here as an option.

**Review by Veckodag — COMMENTED** [review]
Good stuff 👍

**Review by Veckodag — APPROVED** [review]
Good job! 👍

**Comment by Nadhum** [comment]
Thanks for reviewing!

**Comment by Nadhum** [comment]
Ready to merge

**Pull request merged**